sw/device/tests/rstmgr_sw_req_test.c

```diff
@@ -0,0 +1,41 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/base/mmio.h"
+#include "sw/device/lib/dif/dif_rstmgr.h"
+#include "sw/device/lib/runtime/log.h"
+#include "sw/device/lib/testing/check.h"
+#include "sw/device/lib/testing/test_framework/test_main.h"
+
+#include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
+
+static dif_rstmgr_t rstmgr;
+
+const test_config_t kTestConfig;
+
+bool test_main(void) {
+  CHECK_DIF_OK(dif_rstmgr_init(
+      mmio_region_from_addr(TOP_EARLGREY_RSTMGR_AON_BASE_ADDR), &rstmgr));
+
+  dif_rstmgr_reset_info_bitfield_t info;
+  CHECK_DIF_OK(dif_rstmgr_reset_info_get(&rstmgr, &info));
+
+  // If POR, clear reason and request software reset
+  if (info == kDifRstmgrResetInfoPor) {
+    LOG_INFO("Info:POR encountered!");
```

Comment:
Nit: I think you can drop the "Info:" since its already taken care of in the `sw/device/lib/runtime/log.c:stringify_severity()` function which is invoked as a part of invoking LOG_INFO(...) macro.

Reply:
ah sounds good. Will do. Let me check if FATAL has something similar also.

Reply:
Log types are already factored in at the backend (same for DV - these are converted to UVM_INFO|ERROR|FATAL).

Reply:
yes i've removed.